public/index.html

```diff
@@ -36,6 +36,7 @@
       sizes="180x180"
       href="<%= BASE_URL %>img/icons/apple-touch-icon.png"
     />
+    <link rel="stylesheet" href="themes.css">
```

Comment:
What is the reason of having dedicated stylesheet just for themes?

<img width="572" alt="Screenshot 2020-01-08 at 10 15 00" src="https://user-images.githubusercontent.com/6773202/71961310-bb7a5700-31ff-11ea-8057-f77f732d225b.png">

Just add one more `<style></style>` block to the app component.